impurityModel/ed/finite.py

```diff
@@ -287,6 +287,12 @@ def daggerOp(op):
     return opDagger
 
 
+def check_hermitian(op: dict[tuple, int|float|complex]):
+    for process, value in daggerOp(op).items():
+        assert process in op
```

Comment:
a bit unnecessary as op[process] will raise KeyError anyway on the next line.

Reply:
yes. 
I thought maybe it was easier to debug when checks were split up.
But let's remove this line, KeyError is pretty clear.

Reply:
Safe to compare the two dicts with == ? 

Pushed this suggestion.

Reply:
Yes, it's safe!

much neater.

It's always a red flag to name a function check_something as it isn't clear from the name what the return value is. Better call it `assert_hermetian`.

Reply:
renamed function